### PR TITLE
Remove gyb files 🥳

### PR DESCRIPTION
**Pull request by kimdv**

### CODE REVIEW

**Comment by kimdv** [comment]
@swift-ci please test

**Comment by kimdv** [comment]
The Ci might be updated to not add `--gyb-only` as a parameter

**Comment by ahoppen** [comment]
Wohooo 🥳 Nice 

You can remove the `--gyb-only` flag here: Just open a PR for apple/swift and mention it in this PR when requesting CI to trigger cross-PR testing.

https://github.com/apple/swift/blob/8b63ce82730ea60699c86ac7a2d8ea7fcd470f59/utils/swift_build_support/swift_build_support/products/swiftsyntax.py#L82-L88

**Comment by kimdv** [comment]
https://github.com/apple/swift/pull/64201

@swift-ci please test

**Comment by kimdv** [comment]
https://github.com/apple/swift/pull/64201

@swift-ci please test

**Comment by kimdv** [comment]
https://github.com/apple/swift/pull/64201

@swift-ci please test

**Review by ahoppen — APPROVED** [review]
No more gyb 🥳

File: CodeGeneration/Sources/SyntaxSupport/AttributeNodes.swift [hunk 1]
```diff
@@ -0,0 +1,892 @@
+//// Automatically Generated From AttributeNodes.swift.gyb.
+//// Do Not Edit Directly!
```
Comment (hamishknight) [inline CodeGeneration/Sources/SyntaxSupport/AttributeNodes.swift:2]: Presumably these comments are now outdated?

**Comment by DougGregor** [comment]
End of an era, and I love it!

File: CodeGeneration/Sources/SyntaxSupport/AttributeNodes.swift (hunk 1 above)
Comment (kimdv) [reply CodeGeneration/Sources/SyntaxSupport/AttributeNodes.swift:2]: Fixed in https://github.com/apple/swift-syntax/pull/1396

**Pull request merged**